compass-js-frontend/src/main/webapp/ui/view/editor/hierarchy.js

```diff
@@ -118,6 +149,31 @@ var HierarchyView = AmpersandView.extend({
 		// else:
 		//      (ideally, the button would be disabled via a binding)
 		//      complain.
+	},
+	notify: function(type, message) {
```

Comment:
Looks really generic. Should this be in a class of its own, so that it can be reused by other views?